components/BottomNavigation/src/private/MDCBottomNavigationItemView.m

```diff
@@ -98,53 +82,11 @@ - (instancetype)initWithCoder:(NSCoder *)aDecoder {
              @"Unexpected number of subviews. Expected %lu but restored %lu. Unarchiving may fail.",
```

Comment:
Why are we keeping the `-initWithCoder:` code above? I thought we're dropping support for NSCoding?

Reply:
As best I can tell the above code doesn't have to do with our custom property coding/decoding behavior.